scripts/ci/_utils.sh

```diff
@@ -60,6 +60,8 @@ function initialize_common_environment {
     mkdir -p "${AIRFLOW_SOURCES}/files"
     mkdir -p "${AIRFLOW_SOURCES}/dist"
 
+    # shellcheck source=common/_common_values.sh
+    . "${AIRFLOW_SOURCES}/common/_common_values.sh"
```

Comment:
A query (not an issue), can/should we merge all the "common" values in a single script?

- `. "${AIRFLOW_SOURCES}/common/_common_values.sh"`
- `. "${AIRFLOW_SOURCES}/common/_image_variables.sh"`
- `. "${AIRFLOW_SOURCES}/common/_files_for_rebuild_check.sh"`

Reply:
Possibly. I will take a look and change it if I find that it is possible.